english/learn/learn.md

```diff
@@ -0,0 +1,5 @@
+Learning can be very enjoyable.
```

Comment:
In marshmallow.md, there is a line "and [learn](learn/learn.md)", which should reach to this page.